doc/getting-started.rst

```diff
@@ -17,6 +17,35 @@ All amounts are represented in the smallest unit (eg. cents), so USD 5.00 is wri
 
 See :doc:`features/parsing` for additional ways to instantiate a Money object from strings.
 
+Accepted integer values
+-----------------------
```

Comment:
Are you sure this should be in the getting started guide? I would say we cover edge cases with this documentation.